packages/TelegramClient-Tests.package/TCTAcceptanceTests.class/instance/testMessageAuthors.st

```diff
@@ -12,6 +12,6 @@ testMessageAuthors
 	
 	notification := TCUNotification newWithMessage: self receivedMessage.
 	
-	self assert: (self class mockUserFullName = (notification sender fullName)).
-	self assert: (chat_id = (notification chat id)).
-	self assert: (self class mockMessageText = (notification message message))
+	self assert: (notification sender fullName) equals: (self class mockUserFullName).
```

Comment:
should the constant not be the leading value? `self assert: (self class mockUserFullName) equals: (notification sender fullName)`